src/components/Receiving/ItemDetails.js

```diff
@@ -6,62 +6,151 @@ import { get } from 'lodash';
 
 import {
   Button,
+  Checkbox,
+  KeyValue,
   Modal,
+  MultiColumnList,
   Row,
   Select,
   TextField,
-  MultiColumnList,
 } from '@folio/stripes/components';
 
+import css from './ItemDetails.css';
+
+const ITEM_STATUS = {
+  received: 'Received',
+};
+
 class ItemDetails extends Component {
   static propTypes = {
     close: PropTypes.func.isRequired,
     itemList: PropTypes.arrayOf(PropTypes.object).isRequired,
+    locationsOptions: PropTypes.arrayOf(PropTypes.object).isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      allChecked: false,
+      checkedItems: [],
+    };
+  }
+
+  getFooter = (close) => (
+    <Row end="xs">
+      <Button
+        onClick={close}
+      >
+        <FormattedMessage id="ui-orders.receiving.cancelBtn" />
+      </Button>
+      <Button
+        buttonStyle="primary"
+        disabled
+      >
+        <FormattedMessage id="ui-orders.receiving.previousBtn" />
+      </Button>
+      <Button
+        buttonStyle="primary"
+        disabled={!this.state.checkedItems.length}
```

Comment:
you're not consistent at this function. You're passing `close` here from props, but make direct access to `this.state.checkedItems`.
If you ask me, Footer looks like a React SFC to me